tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRTransforms.java

```diff
@@ -61,7 +59,7 @@ public CLDRTransforms setShowProgress(Appendable showProgress) {
     final Set<String> overridden = new HashSet<>();
     final DependencyOrder dependencyOrder = new DependencyOrder();
 
-    static public class RegexFindFilenameFilter implements FilenameFilter {
+    public static class RegexFindFilenameFilter implements FilenameFilter {
```

Comment:
The Google formatter does do consistent reordering of modifiers - I'd forgotten that.
This does mean IDE settings for editing would want to be set to match (e.g. for code templates etc.)